### ISSUE TITLE
Can't open, DLL issue on windows 10

### ISSUE THREAD

**Issue by RadGH** [pbek/loganalyzer#2]
Hey there, I was trying your software out for the first time to look at some access logs (not sure if they'll even work or not!)

I downloaded the most recent zip, extracted it, and tried to run the program. I get some DLL errors. Looking at the installation guide I don't think there are any other steps to follow. Any idea why this is happening?

Screenshots:

https://radleysustaire.com/s3/1926a6/chrome
https://radleysustaire.com/s3/b56afd/
https://radleysustaire.com/s3/4d1b94/
https://radleysustaire.com/s3/dc9dfe/

Thanks

**Comment by pbek** [comment]
It seems the `libstdc++-6.dll`, that is deployed by Qt doesn't work correctly.

Could you please download <https://github.com/pbek/QOwnNotes/blob/develop/appveyor/libstdc%2B%2B-6.dll>, copy it to your LogAnalyzer folder and test if it works.

**Comment by pbek** [comment]
Or even better try the new build: <https://ci.appveyor.com/api/buildjobs/o480hj44cfa8f3je/artifacts/build-systems%2Fappveyor%2FLogAnalyzer.zip>

**Comment by pbek** [comment]
Now also released in <https://github.com/pbek/loganalyzer/releases/tag/1.0.168>

**Comment by pbek** [comment]
btw. how did you find out about LogAnalyzer? :smile:

**Comment by RadGH** [comment]
I appreciate the quick fix! I ended up using GoAccess just for some reporting statistics which was all I needed in the end.

But I gave it another try for you. The new version worked! Unfortunately I tried opening a 300mb apache access log but it crashed shortly after, and attempting to open again immediately closes. Not sure if those types of logs are supported or if the file was too big or what.  But oh well, it was worth a shot!

I found your project by googling something like: **+access_log apache analyzer**

Hope it helps, I don't really need a solution any longer, but hopefully the next person will get more use out of it!
